.github/workflows/release.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     strategy:
       matrix:
         config:
-          - name: "keptn-lifecycle-controller"
```

Comment:
Be aware that this will also push to a new GHCR repo which will not work before pushing an image manually to the new repo.

Reply:
good point ... reverted this change ...